src/main/java/seedu/sudohr/logic/commands/leave/AddEmployeeToLeaveFromToCommand.java

```diff
@@ -0,0 +1,102 @@
+package seedu.sudohr.logic.commands.leave;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.sudohr.logic.commands.Command;
+import seedu.sudohr.logic.commands.CommandResult;
+import seedu.sudohr.logic.commands.exceptions.CommandException;
+import seedu.sudohr.model.Model;
+import seedu.sudohr.model.employee.Employee;
+import seedu.sudohr.model.employee.Id;
+import seedu.sudohr.model.leave.Leave;
+import seedu.sudohr.model.leave.LeaveDate;
+
+/**
+ * Adds a employee using it's displayed index to a specific leave using it's
+ * displayed index in SudoHR.
+ */
+public class AddEmployeeToLeaveFromToCommand extends Command {
+    public static final String COMMAND_WORD = "addEmployeeLeaveFromTo";
+
+    public static final String DATE_CONSTRAINTS = "start date must be before end date and there can "
+            + "only be 7 days difference between start and end date";
```

Comment:
should be 6 days difference right? I think we need to specify clearer in case DG flaw.

I'm thinking we should specify: 
- Constraint: only a maximum of 7 days can be added (i.e maximum 6 days difference between start and end)